notices.json

```diff
@@ -75,8 +75,23 @@
     },
     "localizedMessages": {
       "en": {
-        "title": "Mattermost Boards will be removed on September 28, 2023",
-        "description": "Mattermost Boards is transitioning to a fully community supported plugin. As a result, Boards will be removed from Mattermost Cloud instances on September 28, 2023. Click below to learn more about how this impacts your instance and how to prepare.",
+        "title": "Mattermost Boards will be removed on September 28th, 2023",
+        "description": "Mattermost Boards is transitioning to a fully community supported plugin. As a result, Boards will be removed from Mattermost Cloud instances on September 28th, 2023. Click below to learn more about how this impacts your instance and how to prepare.",
+        "image": "https://raw.githubusercontent.com/mattermost/notices/master/images/Feedback.png",
+        "actionText": "Learn more",
+        "actionParam": "https://forum.mattermost.com/t/upcoming-product-changes-to-boards-and-various-plugins/16669"
+      }
+    }
+  },
+  {
+    "id": "boards_deprecations_end_user",
```

Comment:
@esethna I believe the build is failing because the max character length for id's is 26. The id here is too long.

Reply:
Thanks, good catch, updated